apps/docs/docs/reference/ui/molecules/RadioField.mdx

```diff
@@ -0,0 +1,79 @@
+---
+description: RadioField grouped a Radio input and its corresponding Label.
+sidebar_custom_props:
+  image: https://vtexhelp.vtexassets.com/assets/docs/src/Accordion___66012e5367e86c5b4422ff8d6974b8cd.png
```

Comment:
Need to update this image!

Reply:
Done [here](https://github.com/vtex/faststore/pull/1513/commits/0f7d61eaf73d93e8cab4ad167a17c5142210e35d) :)